db/migrate/20240312122903_make_stripe_payment_intent_generic.rb

```diff
@@ -0,0 +1,43 @@
+# frozen_string_literal: true
+
+class MakeStripePaymentIntentGeneric < ActiveRecord::Migration[7.1]
+  def change
+    change_table :stripe_payment_intents do |t|
+      t.rename :user_id, :initiated_by_id # Make the purpose of the user_id column more clear
+      t.rename :confirmed_by_id, :confirmation_source_id
+      t.rename :confirmed_by_type, :confirmation_source_type
+      t.rename :canceled_by_id, :cancellation_source_id
+      t.rename :canceled_by_type, :cancellation_source_type
+
+      t.string :wca_status
+      t.string :payment_record_type
+      t.integer :payment_record_id
```

Comment:
Nitpick: This should be a b`t.bigint` instead of `integer`, but that won't matter anyways if you just use my renaming suggestion from the other comment

Reply:
Noted!